tck/pom.xml

```diff
@@ -25,6 +25,12 @@
             <type>pom</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We use jboss logger. Does that not work for you ?

Reply:
I see a warning in the logs that there is no slf4 binding

Reply:
mmm, wonder what is pulling in slf4j ? Maybe we can just exclude that dep  from whoever is pulling it in ? (less dependencies is better :) )

Reply:
seems like shrinkwrap-resolver-depchain

Reply:
It's used by shrinkwrap and graphql-java... I assume it's easier to add the jboss-logger-bridge (I'm sure there is one).

Reply:
`slf4j-jboss-logging`

Reply:
fixed

Reply:
Cool !